osCam/core/urls.py

```diff
@@ -8,4 +8,5 @@
 urlpatterns = [
     path('', views.home, name='home'),    
     path('feed/', views.feed, name='feed'),
+    # path('/settings/', views.settings, name='settings'),
```

Comment:
if this path was moved, we can delete the comment.

Reply:
ye sure thing just missed this.